vhosting/components/user.sls

```diff
@@ -8,7 +8,7 @@
 # Ensure user {{ username }} is created
 {{ username }}:
   user.present:
-    - shell: {{ params.get('shell', '/bin/bash') }}
+    - shell: '/bin/bash'
```

Comment:
Jawel Jawel! 
Dit is de vhost-user. Die moet ook op `/bin/false` kunnen als dat nodig is.

En die kan je dus juist met deze wijziging instellen. Ook ZSH, als je vhost dat perse wil ;-)